runtime/android/core/src/org/xwalk/core/XWalkActivity.java

```diff
@@ -128,4 +188,15 @@ protected void onResume() {
         super.onResume();
         mActivityDelegate.onResume();
     }
+
+    private boolean isDownloadMode() {
```

Comment:
Yes. Please take a look at https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview/blob/master/platforms/android/src/org/crosswalk/engine/XWalkWebViewEngine.java

It maybe helpful for future integration into Cordova plugin